spec/system/planning_applications/cancelling_spec.rb

```diff
@@ -0,0 +1,286 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Planning Application Assessment", type: :system do
+  let(:local_authority) {
+    create :local_authority,
+    name: "Cookie authority",
+    signatory_name: "Mr. Biscuit",
+    signatory_job_title: "Lord of BiscuitTown",
+    enquiries_paragraph: "reach us on postcode SW50",
+    email_address: "biscuit@somuchbiscuit.com"
+    }
+  let!(:assessor) { create :user, :assessor, name: "Lorrine Krajcik", local_authority: local_authority }
+  let!(:reviewer) { create :user, :reviewer, name: "Harley Dicki", local_authority: local_authority }
+
+  let!(:planning_application) do
+    create :planning_application, :not_started,
+            :lawfulness_certificate,
+            local_authority: local_authority
+  end
+
+  let(:policy_consideration_1) do
+    create :policy_consideration,
+            policy_question: "The property is",
+            applicant_answer: "a semi detached house"
+  end
+
+  let(:policy_consideration_2) do
+    create :policy_consideration,
+            policy_question: "The project will ___ the internal floor area of the building",
+            applicant_answer: "not alter"
+  end
+
+  let!(:policy_evaluation) do
+    create :policy_evaluation,
+            planning_application: planning_application,
+            policy_considerations: [policy_consideration_1, policy_consideration_2]
+  end
+
+  let!(:drawing) do
+    create :drawing, :with_plan, :proposed_tags,
+           planning_application: planning_application
+  end
```

Comment:
I think drawing, policy_evaluation, policy_consideration_2 and policy_consideration_1 aren't used or needed for these tests, so we can remove them?